group2eperson.md

```diff
@@ -0,0 +1,98 @@
+# Group 2 EPerson mapping Endpoints
```

Comment:
we don't need a new endpoint for that. We should use an association endpoint inside the groups and epersons endpoints, we can eventually decide to implement the mapping only from one side (in such case my preference goes to the groups endpoint as here we will have in any case the "subgroups").

For example to add a subgroup to a group a text-uri/list POST request should be sent to `/api/eperson/groups/<:uuid>/groups`

to add an eperson we should introduce a new link epersons in the groups endpoint and sent a POST request to `/api/eperson/groups/<:uuid>/epersons`
with the request body to be the URI of the subgroup or eperson (i.e. `https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/<:subgroup-uuid>` or `https://dspace7.4science.it/dspace-spring-rest/api/eperson/epersons/<:eperson-uuid>`

A PUT request can be used to fully override the mapping if needed. See #44 for an example

Reply:
This was solved